classes/DepositDAO.inc.php

```diff
@@ -52,23 +52,29 @@ public function getById($depositId, $journalId = null) {
 	 */
 	public function insertObject($deposit) {
```

Comment:
Does it make sense to use more Laravel-like syntax here, like:

```
DB::table('your_table_name')->insert([
    'column1' => 'value1',
    'column2' => 'value2',
    // Add more columns and values as needed
]);
```

Reply:
Definitely! For now, I think it's better to release something, then come back to do the remaining updates.